### PR TITLE
fix(surveys): do not return survey description to client

### PR DESCRIPTION
**Pull request by dmarticus**
## Problem

Fixes the issue described in this bug: https://github.com/PostHog/posthog/issues/23382.  Note that this doesn't solve our problem of oversharing survey data onto the client-side, but it's an easy way to solve for a specific customer painpoint and we can revisit a more complete fix down the line.

## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

New unit test

### CODE REVIEW

File: posthog/api/survey.py [hunk 1]
```diff
@@ -462,6 +462,15 @@ class Meta:
         ]
         read_only_fields = fields
 
+    # We had a user write in complaining that we were exposing the description in the API
+    # (https://posthoghelp.zendesk.com/agent/tickets/15210), which was a problem for them
+    # since they were using it as a way to store sensitive information. Given that we don't ever use
+    # that field to render the survey, we can safely remove it from the API response.
+    def to_representation(self, instance):
```
Comment (Phanatic) [inline posthog/api/survey.py:469]: I'm probably missing something here, but can we get the same result by just removing the `description` property from the serialized properties list here? 

https://github.com/PostHog/posthog/blob/6d48bcff16e2854582e7825bd72bed045bb4ec49/posthog/api/survey.py#L450

**Review by Phanatic — COMMENTED** [review]
I left a comment about the way we remove the property, maybe I'm missing something. 
Otherwise, tests look great, thanks for adding them!

File: posthog/api/survey.py (hunk 1 above)
Comment (dmarticus) [reply posthog/api/survey.py]: you right.  I'm doing that instead.

**Review by Phanatic — APPROVED** [review]
ship it!

**Pull request merged**